lib/modules/dosomething/dosomething_api/resources/member_resource.inc

```diff
@@ -207,6 +207,7 @@ function _member_resource_create($request) {
     'mobile' => !empty($request['mobile']) ? dosomething_user_clean_mobile_number($request['mobile']) : NULL,
     'user_registration_source' => !empty($request['user_registration_source']) ? $request['user_registration_source'] : NULL,
     'northstar_id' => $request['_id'],
+    'created_by_openid_connect' => empty($request['password']) ? 1 : 0, // If no password given, specify that this user *must* login via Northstar.
```

Comment:
[Yes](https://github.com/DoSomething/phoenix/blob/463f43ec8e37009c57a1d41312f0d2d9954d2c39/lib/modules/dosomething/dosomething_user/dosomething_user.module#L1091-L1094)! That's what allows people to have registered via the OpenID Connect flow but still log in if they return to a different computer later and aren't opted-in.